packages/service-worker/worker/test/happy_spec.ts

```diff
@@ -1759,6 +1765,66 @@ describe('Driver', () => {
          });
     });
   });
+
+  describe('freashnessForNavigationUrls', () => {
+    it('doesn\'t create navigate request if when disabled', async () => {
+      const freshnessManifest = createManifestWithFreshness(false);
+      const {server, scope, driver} = initializeSwWith(freshnessManifest);
+
+      await driver.initialized;
+      await makeNavigationRequest(scope, '/', '');
+      await makeNavigationRequest(scope, '/', '');
+      await makeNavigationRequest(scope, '/', '');
+
+      server.assertSawRequestFor('/ngsw.json');
+      server.assertSawRequestFor('/foo.txt');
+      server.assertNoOtherRequests();
+    });
+
+    it('creates the navigation request each time when enabled', async () => {
```

Comment:
I think the following description would be a little more clear:

```suggestion
    it('sends the request to the server when enabled', async () => {
```